releasenotes/notes/fix-vf2-scoring-1q-e2ac29075831d64d.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fix a bug in the :class:`~.VF2Layout` and :class:`~.VF2PostLayout` passes
```

Comment:
We're still doing past tense for these, yeah? `Fixed a bug` might be more consistent.